students/scotchwsplenda/Lesson_9/mailroom_obl_oo/mailroom_oo.py

```diff
@@ -0,0 +1,63 @@
+from operator import itemgetter
+
+
+class Donor():
+
+    def __init__(self, name=None, donations=[]):
+        self.name = name
+        self.donations = donations
+
+    def thanks_mail(self, name, amount):
+        email = ''.join((
+            "\nDear {},\n",
+            "Thanks for the ${:.2f} bucks.\n",
+            "We'll spend it real good baby.\n"
+            )).format(name, amount)
+        return email
+
+
+class DonorCollection(object):
```

Comment:
you should be using your Donor objects in this class, remember that this class represents collection of donor (objects).

a good structure would be to use a dictionary with names as keys and Donor instance as value